sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -838,10 +838,12 @@ public void run() {
                         ErrorLogReport errorLogReport = unprocessedEntry.getValue();
                         if (errorLogReport.report.getThrowable() instanceof NativeException) {
                             Exception exception = errorLogReport.log.getException();
-                            dumpFile = new File(exception.getStackTrace());
-                            exception.setStackTrace(null);
-                            byte[] logfileContents = FileManager.readBytes(dumpFile);
-                            dumpAttachment = ErrorAttachmentLog.attachmentWithBinary(logfileContents, "minidump.dmp", "application/octet-stream");
+                            if (exception.getStackTrace() != null) {
```

Comment:
That does not explain how stack trace is null and seems like a work around.
What is the cause?

Reply:
Also that means we won't send minidump for that crash and thus it will not show up on portal.

Reply:
This will just hide the issue. We should investigate and find root causes of the issue.

Reply:
If we really cannot find the root cause then yes we could avoid the crash, but since we are not yet releasing the SDK we still have time to find why.